.github/workflows/reusable--build-and-tag.yml

```
@@ -163,8 +163,10 @@ jobs:
           if [ "${{ env.WORKFLOW_CONCLUSION }}" == "failure" ];
           then
             echo TEXT="La pose du tag *${{ env.TAG_NAME }}* a échoué :x: Détails sur <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|Github>" >> $GITHUB_ENV
+          elif [ "${{ env.WORKFLOW_CONCLUSION }}" == "skipped" ];
```

Ce ne serait pas plutôt "cancelled" ?

```suggestion
          elif [ "${{ env.WORKFLOW_CONCLUSION }}" == "cancelled" ];
```

---

Tu me mets le doute, j'ai fait le test il me semble et c'était bien skipped je crois mais je refais le test.

---

Je me suis dit que ça serait plus malin de retrouver mon test en fait : https://github.com/pass-culture/pass-culture-main/actions/runs/6761081634/job/18375568593#step:6:3

---

Ah, ok. Peut-être que cette étape-là est "skipped" parce que l'étape d'avant a été "cancelled".